diff --git a/anki/__init__.py b/anki/__init__.py
index 9494669..d185705 100644
--- a/anki/__init__.py
+++ b/anki/__init__.py
@@ -4,8 +4,8 @@
 
 import sys
 
-if sys.version_info[0] < 3 or sys.version_info[1] < 6:
-    raise Exception("Anki requires Python 3.6+")
+if sys.version_info[0] < 3 or sys.version_info[1] < 5:
+    raise Exception("Anki requires Python 3.5+")
 
 if sys.getfilesystemencoding().lower() in ("ascii", "ansi_x3.4-1968"):
     raise Exception("Anki requires a UTF-8 locale.")
diff --git a/aqt/addons.py b/aqt/addons.py
index 4b9cb20..e53adb5 100644
--- a/aqt/addons.py
+++ b/aqt/addons.py
@@ -326,7 +326,7 @@ class AddonsDialog(QDialog):
         if not addon:
             return
         if re.match("^\d+$", addon):
-            openLink(aqt.appShared + f"info/{addon}")
+            openLink(aqt.appShared + "info/%s" % addon)
         else:
             showWarning(_("Add-on was not downloaded from AnkiWeb."))
 
diff --git a/aqt/browser.py b/aqt/browser.py
index 9778abd..b244ece 100644
--- a/aqt/browser.py
+++ b/aqt/browser.py
@@ -1359,7 +1359,7 @@ where id in %s""" % ids2str(sf))
 
         self._updatePreviewButtons()
         self._previewWeb.eval(
-            f"{func}({json.dumps(txt)},'{bodyclass}');")
+            "%s(%s,'%s');" % (func, json.dumps(txt), bodyclass))
 
     def _onPreviewShowBothSides(self, toggle):
         self._previewBothSides = toggle
diff --git a/aqt/editor.py b/aqt/editor.py
index e64f2a0..ef38c46 100644
--- a/aqt/editor.py
+++ b/aqt/editor.py
@@ -81,15 +81,15 @@ class Editor:
         righttopbtns.append(self._addButton('text_clear', 'clear', _("Remove formatting (Ctrl+R)")))
         # The color selection buttons do not use an icon so the HTML must be specified manually
         tip = _("Set foreground colour (F7)")
-        righttopbtns.append(f'''<button tabindex=-1 class=linkb title="{tip}"
+        righttopbtns.append('''<button tabindex=-1 class=linkb title="%s"
             type="button" onclick="pycmd('colour');return false;">
             <div id=forecolor style="display:inline-block; background: #000;border-radius: 5px;"
-            class=topbut></div></button>''')
+            class=topbut></div></button>''' % (tip))
         tip = _("Change colour (F8)")
-        righttopbtns.append(f'''<button tabindex=-1 class=linkb title="{tip}"
+        righttopbtns.append('''<button tabindex=-1 class=linkb title="%s"
             type="button" onclick="pycmd('changeCol');return false;">
             <div style="display:inline-block; border-radius: 5px;"
-            class="topbut rainbow"></div></button>''')
+            class="topbut rainbow"></div></button>''' % (tip))
         righttopbtns.append(self._addButton('text_cloze', 'cloze', _("Cloze deletion (Ctrl+Shift+C)")))
         righttopbtns.append(self._addButton('paperclip', 'attach', _("Attach pictures/audio/video (F3)")))
         righttopbtns.append(self._addButton('media-record', 'record', _("Record audio (F5)")))
diff --git a/aqt/errors.py b/aqt/errors.py
index 4f325d8..080f274 100644
--- a/aqt/errors.py
+++ b/aqt/errors.py
@@ -145,8 +145,8 @@ add-ons section</a> of our support site.
         else:
             platname = "Linux"
 
-        return f"""\
-Anki {appVersion} Python {platform.python_version()} Qt {QT_VERSION_STR} PyQt {PYQT_VERSION_STR}
-Platform: {platname}
-Flags: frz={getattr(sys, "frozen", False)} ao={self.mw.addonManager.dirty}
-"""
+        return """\
+Anki %s Python %s Qt %s PyQt %s
+Platform: %s
+Flags: frz=%s ao=%s
+""" % (appVersion, platform.python_version(), QT_VERSION_STR, PYQT_VERSION_STR, platname, getattr(sys, "frozen", False), self.mw.addonManager.dirty)
diff --git a/aqt/main.py b/aqt/main.py
index 8dc8399..2feacb0 100644
--- a/aqt/main.py
+++ b/aqt/main.py
@@ -288,7 +288,7 @@ close the profile or restart Anki."""))
                 if getattr(w, "silentlyClose", None):
                     w.close()
                 else:
-                    showWarning(f"Window should have been closed: {w}")
+                    showWarning("Window should have been closed: %s" % w)
 
     def unloadProfileAndExit(self):
         self.unloadProfile(self.cleanupAndExit)
diff --git a/aqt/reviewer.py b/aqt/reviewer.py
index f87b1be..8e54929 100644
--- a/aqt/reviewer.py
+++ b/aqt/reviewer.py
@@ -119,12 +119,12 @@ class Reviewer:
 
     def revHtml(self):
         extra = self.mw.col.conf.get("reviewExtra", "")
-        return f"""
+        return """
 <div id=_mark>&#x2605;</div>
 <div id=_flag>&#x2691;</div>
 <div id=qa></div>
-{extra}
-"""
+%s
+""" % extra
 
     def _initWeb(self):
         self._reps = 0
diff --git a/aqt/utils.py b/aqt/utils.py
index 2fdf895..4ab040a 100644
--- a/aqt/utils.py
+++ b/aqt/utils.py
@@ -503,7 +503,7 @@ class MenuList:
         end = items[-1].title
         prefix = os.path.commonprefix([start.upper(), end.upper()])
         n = len(prefix)+1
-        return f"{start[:n].upper()}-{end[:n].upper()}"
+        return "%s-%s" % (start[:n].upper(), end[:n].upper())
 
 class SubMenu(MenuList):
     def __init__(self, title):
diff --git a/aqt/webview.py b/aqt/webview.py
index 5cc70cd..d8c7c6c 100644
--- a/aqt/webview.py
+++ b/aqt/webview.py
@@ -213,21 +213,21 @@ border-radius:5px; font-family: Helvetica }"""
         from aqt import mw
         head =  mw.baseHTML() + head + csstxt + jstxt
 
-        html=f"""
+        html="""
 <!doctype html>
 <html><head>
-<title>{self.title}</title>
+<title>%s</title>
 
 <style>
-body {{ zoom: {self.zoomFactor()}; {fontspec} }}
-{buttonspec}
+body {{ zoom: %s; %s }}
+%s
 </style>
   
-{head}
+%s
 </head>
 
-<body>{body}</body>
-</html>"""
+<body>%s</body>
+</html>""" % (self.title, self.zoomFactor(), fontspec, buttonspec, head, body)
         #print(html)
         self.setHtml(html)
 
